setup.py

```diff
@@ -47,53 +46,50 @@ def read(fname):
     'singledispatchmethod; python_version < "3.8.0"',
     'dataclasses; python_version < "3.7.0"',
     'PyYAML>=5.1',
-    'orjson>=3.6.1',    # this is a faster json lib support inf/nan and datetime
-                        # v3.6.1 is the last version supports Python 3.6
+    'orjson>=3.6.1',  # this is a faster json lib support inf/nan and datetime
+    # v3.6.1 is the last version supports Python 3.6
 ]
 
 # extra requirements for biothings.web
 web_extra_requires = [
-    'msgpack>=0.6.1',   # support format=msgpack
+    'msgpack>=0.6.1',  # support format=msgpack
     'sentry-sdk>=1.5.3',  # new sentry package
 ]
 
 # extra requirements to run biothings.hub
 hub_requires = [
-    'beautifulsoup4',   # used in dumper.GoogleDriveDumper
-    'aiocron==1.8',     # setup scheduled jobs
-    'aiohttp==3.8.1',   # elasticsearch requires aiohttp>=3,<4
+    'beautifulsoup4',  # used in dumper.GoogleDriveDumper
+    'aiocron==1.8',  # setup scheduled jobs
+    'aiohttp==3.8.1',  # elasticsearch requires aiohttp>=3,<4
```

Comment:
This is a case I'm kind of like the the old style better. Wonder if have a way to control this behavior.

But also it's not a big deal of course.